L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -43,7 +73,14 @@ HGCalConcentratorSuperTriggerCellImpl::getSuperTriggerCellId(int detid) const {
     else{
       int TC_wafer = TC_idV8.wafer();
       int thickness = triggerTools_.thicknessIndex(detid,true);
-      int TC_split = ( TC_idV8.cell() & kSplit_.at( stcSize_.at(thickness) ) );
+      int TC_split = -1;
+      if ( STCsize > 0 ){
```

Comment:
The reason for this is that coarse TCs are always of size 2, and I wanted to reuse the same superTC code for selection. Probably there's a better way.

Reply:
Super TCs are a type of coarse TCs and super TCs can have sizes larger than 2. There shouldn't be conceptual differences in the treatment of "super TCs" and "coarse TCs".

Reply:
Right, since they are conceptually the same I wanted to use the same code. Currently in the config file the size of the "super TC" is set by the thickness of the wafer.
However in the thick layers we also need "coarse TCs" (always of size 2).